Source/InitialConditions/BlackHoles/TwoPuncturesInitialData.impl.hpp

```diff
@@ -0,0 +1,88 @@
+/* GRChombo
+ * Copyright 2012 The GRChombo collaboration.
+ * Please refer to LICENSE in GRChombo's root directory.
+ */
+
+#if !defined(TWOPUNCTURESINITIALDATA_HPP_)
+#error "This file should only be included through TwoPuncturesInitialData.hpp"
+#endif
+
+#ifndef TWOPUNCTURESINITIALDATA_IMPL_HPP_
+#define TWOPUNCTURESINITIALDATA_IMPL_HPP_
+
+void TwoPuncturesInitialData::compute(Cell<double> current_cell) const
+{
+    Vars<double> vars;
+    // Set only the non-zero components explicitly below
+    VarsTools::assign(vars, 0.);
+
+    Coordinates<double> coords(current_cell, m_dx, m_center);
+    Tensor<2, double> h_phys, extrinsic_K;
```

Comment:
Maybe make extrinsic_K -> K_tensor which matches the Weyl class naming? To be consistent maybe we should be using K_phys, but either is ok for me.
I was also wondering about using gamma for the physical metric, but h_phys is ok for me, I don't think we have named this otherwise elsewhere.

Reply:
I guess `K_phys` might be mistaken for the trace of the extrinsic curvature. I'm happy with `K_tensor`. `h_phys` is definitely consistent with what we do elsewhere.